MP3 Tag Editor/Mp3TagsManager.cs

```diff
@@ -0,0 +1,166 @@
+        var words = Regex.Split(stringTag, @"\W+");
+
+        for (var i = 0; i < words.Length; i++)
+        {
+            if (!string.IsNullOrEmpty(words[i]))
+                words[i] = textInfo.ToTitleCase(words[i].ToLower());
+        }
+        
+        stringTag = string.Join("", words);
+        Enum.TryParse(stringTag, out Mp3Tag tag);
+        
+        return ModifyMp3Tags(files, tag, value);
+    }
+
+
+    /// <summary>
+    /// Modifies the specified MP3 tags in the given file or directory.
+    /// </summary>
+    /// <param name="files">The list of MP3 files to modify.</param>
+    /// <param name="tag">The MP3 tag to modify.</param>
+    /// <param name="value">The new value to assign to the MP3 tag.</param>
+    /// <exception cref="InvalidMp3TagException">Thrown when the specified MP3 tag does not exist.</exception>
+    /// <exception cref="InvalidMp3TagValueException">Thrown when the provided value is invalid for the specified MP3 tag.</exception>
+    /// <returns>A list of modified mp3 files.</returns>
+    public static IEnumerable<TagLib.File> ModifyMp3Tags(IEnumerable<TagLib.File> files, Mp3Tag tag, dynamic value)
+    {
+        var mp3Files = files.ToList();
+        mp3Files.ForEach(file => ModifyMp3Tag(file, tag, value));
+        return mp3Files;
+    }
+
+    /// <summary>
+    /// Modifies the specified MP3 tag of a TagLib.File.
+    /// </summary>
+    /// <param name="file">The TagLib.File object to modify.</param>
+    /// <param name="mp3Tag">The MP3 tag to modify.</param>
+    /// <param name="value">The new value to assign to the MP3 tag.</param>
+    /// <returns>The modified TagLib.File object.</returns>
+    /// <exception cref="InvalidMp3TagException">Thrown when the specified MP3 tag does not exist.</exception>
+    /// <exception cref="InvalidMp3TagValueException">Thrown when the provided value is invalid for the specified MP3 tag.</exception>
+    private static void ModifyMp3Tag(TagLib.File file, Mp3Tag mp3Tag, dynamic value)
+    {
+        if (_tagLibTagTypes is null)
+        {
+            InitializeTagLibTagTypesList();
+        }
+
+        var propertyInfo = typeof(Tag).GetProperty(mp3Tag.ToString());
+
+        if (propertyInfo is null)
+        {
+            throw new InvalidMp3TagException(file);
+        }
+
+        try
+        {
+            var tagType = _tagLibTagTypes?[mp3Tag.ToString()];
+            ModifyTagProperty(file, propertyInfo, tagType, value);
+        }
+        catch (Exception e)
+        {
+            HandlePropertyModificationException(e, mp3Tag, value);
+        }
+    }
+
+    /// <summary>
+    /// Initializes the dictionary of tag property names and their associated types.
+    /// </summary>
+    private static void InitializeTagLibTagTypesList()
+    {
+        _tagLibTagTypes = new Dictionary<string, Type>();
+        var file = TagLib.File.Create(Path.Combine(Directory.GetCurrentDirectory(), @"Resources\sample.mp3"));
```

Comment:
The file should be renamed to a name that clearly shows that it's not to be tampered with
Also if we're going to use it as part of our code we should use something that's not copyrighted lmao